src/PackageDependencyViewExtension/PackageDependencyView.xaml

```diff
@@ -0,0 +1,91 @@
+﻿<UserControl x:Class="Dynamo.PackageDependency.PackageDependencyView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             mc:Ignorable="d" 
+             VerticalAlignment="Top"
+             HorizontalAlignment="Left">
+    <Grid Background= "#353535" Height="800">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <DataGrid 
+            Name="PackageDependencyTable" 
+            HorizontalAlignment="Left"
+            VerticalAlignment="Top" 
+            Width="500"
+            Margin="5,5,0,0" 
```

Comment:
@QilongTang minor comment, this margin looks odd for some reason. There is already a white border so do you think we can get rid of this?

Reply:
Thanks for reminding me! Removed, I noticed you have built-in margin already